test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -1243,6 +1243,21 @@ spec setup =
             t = table_builder [["x", ["1", "2", "3"]]]
             t.at "x" . const (t.at "x") . should_fail_with Illegal_Argument
 
+    Test.group prefix+"Table.make_constant_column" <|
+        Test.specify "Should allow the creation of constant columns" <|
+            t = table_builder [["x", ["1", "2", "3"]]]
+            t.make_constant_column True . to_vector . should_equal [True, True, True]
+            t.make_constant_column 12 . to_vector . should_equal [12, 12, 12]
+            t.make_constant_column 12.3 . to_vector . should_equal [12.3, 12.3, 12.3]
+            t.make_constant_column "asdf" . to_vector . should_equal ["asdf", "asdf", "asdf"]
+
+        Test.specify "Should allow the creation of constant columns on a table with no rows" <|
+            t = table_builder [["x", ["1", "2", "3"]]]
+            empty = t.take 0
+            constant = empty.make_constant_column 42
+            empty2 = empty.set constant
+            empty2.column_names.should_equal ['x', '42']
```

Comment:
We need tests that check that, when the table we work on is empty, the type of the column is right.

I.e. we need something like
```
empty.at "42" . value_type . should_equal Value_Type.Integer
```

and to make sure `Integer` is not just hardcoded there, we need some other cases as well:
```
empty.make_constant_column "foo" . value_type . should_equal Value_Type.Char
empty.make_constant_column False . value_type . should_equal Value_Type.Boolean
empty.make_constant_column 1.0 . value_type . should_equal Value_Type.Float
```

Reply:
Done